graph/nosql/mongo/mongo.go

```diff
@@ -64,8 +64,12 @@ func dialDB(addr string, opt graph.Options) (*DB, error) {
 	if err != nil {
 		return nil, err
 	}
+	dbName, err := options.StringKey("database_name", nosql.DefaultDBName)
```

Comment:
Since setting the database name in `dialMongo`does not change it, could you please remove it from there?